interfaces/builtin/network_control.go

```diff
@@ -101,7 +101,11 @@ network sna,
 /{,usr/}{,s}bin/arpd ixr,
 /{,usr/}{,s}bin/bridge ixr,
 /{,usr/}{,s}bin/dhclient Pxr,             # use ixr instead if want to limit to snap dirs
+/{,usr/}{,s}bin/dhclient-script ixr,
 /{,usr/}{,s}bin/ifconfig ixr,
+/{,usr/}{,s}bin/ifdown ixr,
+/{,usr/}{,s}bin/ifquery ixr,
+/{,usr/}{,s}bin/ifup ixr,
```

Comment:
I think at one point we actively decided that ifupdown would not be supported, but that was a long time ago before classic distro was really a thing, so it makes sense to support it now.